edx_lint/files/pylintrc

```diff
@@ -36,28 +249,93 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
```

Comment:
This is less about increase/decrease in running speed, and more about checks that we frequently add exceptions to.